tinkerers/sachin_raj_m.md

```diff
@@ -0,0 +1,15 @@
+# Sachin Raj M
+## 
+
+<img src="https://raw.githubusercontent.com/Trajectory-Bootcamp/Trajectory-GitHub-Task/main/images/hecker.jpg">
```

Comment:
Heyy Sachin, if you wish to include an image in your markdown file please add your own image to the pull request

Reply:
Yeah, I just removed the img tag